Common/include/containers/container_decorators.hpp

```diff
@@ -105,42 +157,11 @@ class C3DContainerDecorator {
 };
 
 /*!
- * \brief Some typedefs for the
+ * \brief Some typedefs for 3D containers
  */
 using C3DIntMatrix = C3DContainerDecorator<su2matrix<unsigned long> >;
 using C3DDoubleMatrix = C3DContainerDecorator<su2activematrix>;
-
-/*!
- * \class CVectorOfMatrix
- * \brief This contrived container is used to store small matrices in a contiguous manner
- *        but still present the "su2double**" interface to the outside world.
- *        The "interface" part should be replaced by something more efficient, e.g. a "matrix view".
- */
-class CVectorOfMatrix: public C3DDoubleMatrix {
-private:
-  su2matrix<Scalar*> interface;
```

Comment:
The main motivation for this change is to store "vectors of matrices" i.e. gradients in a contiguous way without needing an extra matrix of pointers to provide the `su2double**` compatibility with other areas of the code.